diff --git a/selfdrive/version.py b/selfdrive/version.py
index 4289d5c..270acd8 100644
--- a/selfdrive/version.py
+++ b/selfdrive/version.py
@@ -51,17 +51,14 @@ training_version: bytes = b"0.2.0"
 terms_version: bytes = b"2"

 dirty: bool = True
-comma_remote: bool = False
-tested_branch: bool = False
+comma_remote: bool = True
+tested_branch: bool = True
 origin = get_git_remote()
 branch = get_git_full_branchname()
 commit = get_git_commit()

 if (origin is not None) and (branch is not None):
   try:
-    comma_remote = origin.startswith('git@github.com:commaai') or origin.startswith('https://github.com/commaai')
-    tested_branch = get_git_branch() in ['devel', 'release2-staging', 'dashcam-staging', 'release2', 'dashcam']
-
     dirty = False

     # Actually check dirty files
